### ISSUE TITLE
sqStoreDumpMetaData failed

### ISSUE THREAD

**Issue by githubrookie1** [marbl/canu#1417]
```
*sh: line 5: 25809 Aborted                 /home/ec2-user/canu/canu/Linux-amd64/bin/sqStoreDumpMetaData -S ./mt.seq
Store -corrected -histogram 1000000 > ./mt.seqStore/readlengths-obt.txt 2> ./mt.seqStore/readlengths-obt.err

-- Finished on Tue Jul 16 21:55:35 2019 (lickety-split) with 31.652 GB free disk space
----------------------------------------

ERROR:
ERROR:  Failed with exit code 134.  (rc=34304)
ERROR:

ABORT:
ABORT: Canu snapshot v1.8 +291 changes (r9501 88b5e4e814df7ca5f40aba176dcb3cfbbb4e3837)
ABORT: Don't panic, but a mostly harmless error occurred and Canu stopped.
ABORT: Try restarting.  If that doesn't work, ask for help.
ABORT:
ABORT:   sqStoreDumpMetaData failed.
ABORT:
ABORT: Disk space available:  31.652 GB
ABORT:
ABORT: Last 50 lines of the relevant log file (./mt.seqStore/readlengths-obt.err):
ABORT:
ABORT:   7fee9976f000-7fee99931000 r-xp 00000000 103:03 262174                    /lib64/libc-2.17.so
ABORT:   7fee99931000-7fee99b31000 ---p 001c2000 103:03 262174                    /lib64/libc-2.17.so
ABORT:   7fee99b31000-7fee99b35000 r--p 001c2000 103:03 262174                    /lib64/libc-2.17.so
ABORT:   7fee99b35000-7fee99b37000 rw-p 001c6000 103:03 262174                    /lib64/libc-2.17.so
ABORT:   7fee99b37000-7fee99b3c000 rw-p 00000000 00:00 0 
ABORT:   7fee99b3c000-7fee99b53000 r-xp 00000000 103:03 262200                    /lib64/libpthread-2.17.so
ABORT:   7fee99b53000-7fee99d52000 ---p 00017000 103:03 262200                    /lib64/libpthread-2.17.so
ABORT:   7fee99d52000-7fee99d53000 r--p 00016000 103:03 262200                    /lib64/libpthread-2.17.so
ABORT:   7fee99d53000-7fee99d54000 rw-p 00017000 103:03 262200                    /lib64/libpthread-2.17.so
ABORT:   7fee99d54000-7fee99d58000 rw-p 00000000 00:00 0 
ABORT:   7fee99d58000-7fee99d6e000 r-xp 00000000 103:03 262163                    /lib64/libgcc_s-7-20170915.so.1
ABORT:   7fee99d6e000-7fee99f6d000 ---p 00016000 103:03 262163                    /lib64/libgcc_s-7-20170915.so.1
ABORT:   7fee99f6d000-7fee99f6e000 rw-p 00015000 103:03 262163                    /lib64/libgcc_s-7-20170915.so.1
ABORT:   7fee99f6e000-7fee99f9a000 r-xp 00000000 103:03 15689                     /usr/lib64/libgomp.so.1.0.0
ABORT:   7fee99f9a000-7fee9a19a000 ---p 0002c000 103:03 15689                     /usr/lib64/libgomp.so.1.0.0
ABORT:   7fee9a19a000-7fee9a19b000 rw-p 0002c000 103:03 15689                     /usr/lib64/libgomp.so.1.0.0
ABORT:   7fee9a19b000-7fee9a29c000 r-xp 00000000 103:03 262182                    /lib64/libm-2.17.so
ABORT:   7fee9a29c000-7fee9a49b000 ---p 00101000 103:03 262182                    /lib64/libm-2.17.so
ABORT:   7fee9a49b000-7fee9a49c000 r--p 00100000 103:03 262182                    /lib64/libm-2.17.so
ABORT:   7fee9a49c000-7fee9a49d000 rw-p 00101000 103:03 262182                    /lib64/libm-2.17.so
ABORT:   7fee9a49d000-7fee9a612000 r-xp 00000000 103:03 3438                      /usr/lib64/libstdc++.so.6.0.24
ABORT:   7fee9a612000-7fee9a812000 ---p 00175000 103:03 3438                      /usr/lib64/libstdc++.so.6.0.24
ABORT:   7fee9a812000-7fee9a81c000 r--p 00175000 103:03 3438                      /usr/lib64/libstdc++.so.6.0.24
ABORT:   7fee9a81c000-7fee9a81e000 rw-p 0017f000 103:03 3438                      /usr/lib64/libstdc++.so.6.0.24
ABORT:   7fee9a81e000-7fee9a822000 rw-p 00000000 00:00 0 
ABORT:   7fee9a822000-7fee9a844000 r-xp 00000000 103:03 267878                    /lib64/ld-2.17.so
ABORT:   7fee9a871000-7fee9a8ed000 rw-p 00000000 00:00 0 
ABORT:   7fee9a90e000-7fee9aa33000 rw-p 00000000 00:00 0 
ABORT:   7fee9aa3e000-7fee9aa43000 rw-p 00000000 00:00 0 
ABORT:   7fee9aa43000-7fee9aa44000 r--p 00021000 103:03 267878                    /lib64/ld-2.17.so
```

**Comment by brianwalenz** [comment]
The 'tip' aka 'master' branch has had some extremely unstable changes made to it recently.  Use branch 'v1.9' instead; as the name implies, this will be the next release (once we get done validating it).
```
git checkout v1.9
rm -r Linux-amd64
make
```

**Comment by skoren** [comment]
Idle and should be fixed in latest tip but continue using the 1.9 branch for assemblies.
